### DISCUSSION TITLE
Can room-assistant be installed on a Raspberry Pi 1 or 2?

### DISCUSSION THREAD

**Discussion by sulsa** [mKeRix/room-assistant#349] [Q&A]
I see the requirements say that a zero W, 3 or 4 are needed, however I would think a Raspberry Pi 2's specs would exceed those of a zero. I have a few of those laying around. Seems like a great way to put them to use.
Thanks!

**Comment by JesusRo** [comment]
I got a Raspberry Pi2 with a Bluethooth USB dongle working fine on the cluster.
So far,
- Pi 4, hassio plugin
- Pi 3 x2
- Pi 2 with USB dongle 
- Pi Zero W
and didn´t notice anything diferent between them relative to performance or reliability
Nonetheless, I use bluetoothLowEnergy and xiaomiMi which both are passive let´s say. No idea about bluetoothClassic to be honest

**Comment by mKeRix** [comment]
room-assistant will run on pretty much any Linux device, given it has the hardware capabilities for the integrations you want. Those can also be added though, so for the Pi 2 you could e.g. plug in a Bluetooth dongle.

In fact you can run room-assistant on any platform that supports NodeJS, although then not all integrations will work and some will behave differently than on Linux.

**Comment by zuk** [comment]
For anyone ending up here in 2022+, you need Node 16 or later for current versions of Room Assistant. Node 16 is not officially supported on Raspberry Pi 1/2, but you can install it directly from https://unofficial-builds.nodejs.org/download/release/v16.18.1/node-v16.18.1-linux-armv6l.tar.gz and it seems to work fine. Just copy the contents of that archive into `/usr/local/`.

As mentioned above, you'll also need a USB Bluetooth dongle since Pi 1/2 doesn't have built in bluetooth
